bugwarrior/docs/services/phabricator.rst

```diff
@@ -60,6 +60,34 @@ you can find it out by querying Phabricator Conduit
 the methods which return the needed info are ``user.query``, ``project.query``
 and ``repository.query`` respectively.
 
+Import Projects as Tags
++++++++++++++++++++++++
+
+Phabricator tasks may belong to multiple projects, and differentials (PRs)
+belong to a diffusion repository which may belong to multiple projects; to
+use those projects' slugs (short identifying, human readable strings) as
+tags, you can use the ``phabricator.should_set_tags`` option::
+
+    phabricator.should_set_tags = True
```

Comment:
I wonder if `import_projects_as_tags` (in the spirit of github's `import_labels_as_tags`) would be more clear since there are other uses of projects.